Spigot-Server-Patches/0248-Send-attack-SoundEffects-only-to-players-who-can-see.patch

```diff
@@ -0,0 +1,76 @@
+From 765ddf65fae7e423c11b310b816e8674c7a45bf8 Mon Sep 17 00:00:00 2001
+From: Brokkonaut <hannos17@gmx.de>
+Date: Tue, 31 Oct 2017 03:26:18 +0100
+Subject: [PATCH] Send attack SoundEffects only to players who can see the
+ attacker
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityHuman.java b/src/main/java/net/minecraft/server/EntityHuman.java
+index 00fe92f..1b36d8c 100644
+--- a/src/main/java/net/minecraft/server/EntityHuman.java
++++ b/src/main/java/net/minecraft/server/EntityHuman.java
+@@ -939,6 +939,15 @@ public abstract class EntityHuman extends EntityLiving {
+         this.j = 0;
+     }
+ 
++    // Paper start - helper method for attack(Entity). Send SoundEffect to everyone who can see this entity
++    private void sendAttackSoundEffect(SoundEffect effect) {
```

Comment:
Please rename this to sendSoundEffect, since it can be used for any sound effect and we potentially will have more places to use this for.